CMakeLists.txt

```diff
@@ -1,11 +1,144 @@
 cmake_minimum_required(VERSION 3.9)
-project(readerwriterqueue VERSION 1.0.0)
+set(CMAKE_CXX_STANDARD 11)  # don't need to specify manually per target anymore.
```

Comment:
`ReaderWriterQueue` supports C++03 on certain platforms.

Reply:
Can you specify in exactly what scenarios this is true?  If you're talking about compilers supporting individual C++11 features with out supporting C++11, Cmake supports specifying individual flags for C++ features, If I know those, I can change the target to instead depend on those c++ features. see https://cmake.org/cmake/help/latest/manual/cmake-compile-features.7.html and https://cmake.org/cmake/help/latest/command/target_compile_features.html#command:target_compile_features

Reply:
Mostly VC++2010. I guess people using that are unlikely to be using CMake anyway.